datumaro/plugins/cvat_format/converter.py

```diff
@@ -276,17 +351,33 @@ def _get_label_attrs(self, label):
             label = label_cat[label]
         return set(chain(label.attributes, label_cat.attributes)) - self._context._builtin_attrs
 
-    def _write_shape(self, shape, item):
-        if shape.label is None:
-            log.warning("Item %s: skipping a %s with no label", item.id, shape.type.name)
+    def _write_shape(self, shape, item=None, write_label_info=True, write_frame=False):
+        item_id = "None" if item is None else item.id
+
+        if write_label_info and shape.label is None:
+            log.warning("Item %s: skipping a %s with no label", item_id, shape.type.name)
             return
 
-        label_name = self._get_label(shape.label).name
-        shape_data = OrderedDict(
-            [
-                ("label", label_name),
-                ("occluded", str(int(shape.attributes.get("occluded", False)))),
-            ]
+        if write_frame and "frame" not in shape.attributes:
+            log.warning("Skipping a %s with no frame", shape.type.name)
+            return
+
+        if write_label_info:
+            label_name = self._get_label(shape.label).name
+            shape_data = OrderedDict([("label", label_name)])
+        else:
+            shape_data = OrderedDict()
+
+        if write_frame:
+            shape_data.update(OrderedDict([("frame", str(int(shape.attributes.get("frame", 0))))]))
+        shape_data.update(
+            OrderedDict([("occluded", str(int(shape.attributes.get("occluded", False))))])
+        )
+        shape_data.update(
+            OrderedDict([("outside", str(int(shape.attributes.get("outside", False))))])
+        )
+        shape_data.update(
+            OrderedDict([("keyframe", str(int(shape.attributes.get("keyframe", 0))))])
```

Comment:
In here, `keyframe` could be `int` but in `test_video_export`, `keyframe` is set as `True`. Is both possible? I'm asking because I think it should be a number like `int`.

Reply:
Good point! I'll change the default value to False.